specs/prometheus/alert_manager/alert_manager.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  name: stork #This name is important since the Alertmanager pods wont start unless a secret named alertmanager-${ALERTMANAGER_NAME} is created. in this case if would expect alertmanager-portworx secret in the kube-system namespace
```

```suggestion
  name: stork #This name is important since the Alertmanager pods won't start unless a secret named alertmanager-${ALERTMANAGER_NAME} is created. in this case, if would expect alertmanager-stork secret in the kube-system namespace
```
That comment is talking about a secret, did you mean service?

---

@ram-infrac Does this need to be updated?